[DOCS-352] Getting Started guide: users land on the Assets page (#264)
cobalthq/cobalt-product-public-docs: content/en/Getting started/Assets/_index.md

---
title: "Define Your Assets"
linkTitle: "Define Your Assets"
weight: 10
description: >
  Security professionals perform pentests on your assets. Collect the info they need.
---

{{% pageinfo %}}
You can launch multiple pentests for an [asset](/getting-started/glossary/#asset). Once you've set up an asset, you can reuse it in your next pentests.
{{% /pageinfo %}}

![Workflow for creating a pentest](/gsg/CreatePentestFlowStage1.png "Workflow for creating a pentest")
<br>

Once you've [signed in](/getting-started/sign-in/), you land on the **Assets** page.

- To add a single asset, select **New Asset**. Specify [asset details](#asset-details).

> - To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).

Reviewer: Do we specify how to format uploaded assets in these spreadsheet formats? Based on the description and the **Asset Details** section, I might conclude:

```csv
<asset title>, <asset type>, <asset description>
<asset title>, <asset type>, <asset description>
...
```

Author: @mjang-cobalt In the overlay for uploading assets, users download a template where they can see the format.

Author: @mjang-cobalt With Danny's help, I updated the images displaying the pentest creation workflow.

Reviewer: I'm going to "Resolve" this comment and suggest a new change, based on the "Assets_Upload_Template.xlsx" file from our app.

> - To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).
>   - If the upload is successful, all your assets from the file are added. Otherwise, no assets are created. Our algorithm doesn't process the request partially.
>   - We don't prevent you from creating duplicate assets.

Reviewer: - I'm suggesting that we include the Assets_Upload_Template.xlsx file to the `/static/gsg/` directory
- IMO, we don't need to specify what happens if the upload fails

```suggestion
- To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. To help, we have an [Assets Upload Template](/gsg/Assets_Upload_Template.xlsx).
   
   Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).
  - If the upload is successful, all your assets from the file are added.
  - We don't prevent you from creating duplicate assets.
```

Author: @mjang-cobalt The procedure describing how to upload assets in bulk was addressed in https://github.com/cobalthq/cobalt-product-public-docs/pull/203

The wording reflects exactly what Yogi requested back then. I don't think we should upload a template to Product Docs. The template may change. The SSoT is the template available in the UI.

Reviewer: Thank you, I had forgotten that PR.
